Assets/HoloToolkit/Utilities/Scripts/HeadsUpDirectionIndicator.cs

```diff
@@ -100,16 +101,17 @@ private void Update()
             }
             else
             {
+                Camera mainCamera = CameraCache.main;
```

Comment:
Not really for caching but I'm just used to have a local reference when I use something more than once. Using the full call to reference it every time and sometimes even twice in the same line can make the code quite a bit longer. I could change it back in cases where it's not as bad.